docs/sources/integrations/inbound-email/index.md

```diff
@@ -22,14 +22,14 @@ You must have an Admin role to create integrations in Grafana OnCall.
 
 1. In the **Integrations** tab, click **+ New integration to receive alerts**.
 2. Select **Inbound Email** from the list of available integrations.
-3. Get your dedicated email address in the **How to connect** window.
+3. Get your dedicated email address in the **Integration email** section and use it to send your emails.
```

Comment:
"use it to send your emails" as in like send your alert notifications to this address? 

```suggestion
3. Get your dedicated email address in the **Integration email** section and use it to send your emails.
```